src/Stan/Inspection.hs

```diff
@@ -45,3 +47,21 @@ data Severity
 -- | Show 'Inspection' in a human-friendly format.
 prettyShowInspection :: Inspection -> Text
 prettyShowInspection = show
+
+{- | List of all inspections.
+-}
+inspections :: [Inspection]
+inspections =
+    [ Inspection
+        { inspectionId = Id "HEAD"  -- TODO: we need to come up with naming scheme for inspection ids
+        , inspectionName = "Partial 'head'"  -- TODO: does it make sense?
```

Comment:
Maybe with semicolon `Partial: head`? And should we add `base/head`?

Reply:
Excellent idea :ok_hand: